ledger-service/http-json-cli/src/main/scala/com/daml/http/dbbackend/DBConfig.scala

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http.dbbackend
+
+import com.typesafe.scalalogging.StrictLogging
+import scalaz.std.either._
+import scalaz.std.option._
+import scalaz.syntax.traverse._
+import scalaz.{Show, StateT}
+
+import com.daml.dbutils, dbutils.DBConfig
+
+private[http] final case class JdbcConfig private (
+    baseConfig: dbutils.JdbcConfig,
+    createSchema: Boolean = false,
```

Comment:
Didn't I remove this field? The field `dbStartupMode` is the replacement and we currently have both fields, can you check and if so remove that field (again)?

Reply:
I don't want to do the archaeology here, only to say that my `merge-base` still has this field. Anyway, removed in 854a13bd345